R/df_to_exb.r

```diff
@@ -0,0 +1,340 @@
+#' Dataframe to .exb
+#'
+#' Convert a dataframe to Exmaralda .exb
+#'
+#' @author Valeria Buntiakova <valleriabun@gmail.com>
+#'
+#' @param df an R dataframe object that contains columns named 'tier', 'tier_name', 'content', 'time_start', 'time_end' and 'id'
+#' @param name transcription name
+#' @param outputPath filepath
+#' @param referenced_file a filepath for .wav
+#' @param ud_meta a vector ('key':'value') of meta information
+#' @param speaker_table a table with speaker information; must include columns 'id', 'abbreviation', 'sex'
```

Comment:
Is it possible to calculate it from the df?

Reply:
Calculating id is possible in all cases, i think; abbreviation - only if the df we are working with has been received from a .exb file in the first place. Everything else is not in the df. The best way of adding a speaker table would be to open Partitur Editor, which has a fitting interface for it, and do it manually, but we decided that our function should be able to do it.